packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -180,7 +180,7 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
                 <WalletText color='general' size='sm'>
                     {transaction.status_name}
                 </WalletText>
-                {!isMobile && transaction.is_valid_to_cancel && (
+                {!isMobile && !!transaction.is_valid_to_cancel && (
```

Comment:
just wondering whats the difference (both versions, old and new seems to be doing the same?)

Reply:
Hi @wojciech-deriv,
`typeof transaction.is_valid_to_cancel = 0 | 1 | undefined`
The value for `transaction.is_valid_to_cancel` is considered as it is if we don't explicitly transform it to boolean using `!!`.
That's the reason we were seeing `0` after the status name.